tests/sqlparser_snowflake.rs

```diff
@@ -2256,3 +2256,9 @@ fn asof_joins() {
           "ORDER BY s.observed",
     ));
 }
+
+#[test]
+fn test() {
```

Comment:
```suggestion
fn test_parse_position() {
```
or something similar?

Reply:
lol, thanks, forgot to go back and update it :sweat_smile: